lib/exception_handling.rb

```diff
@@ -132,7 +131,7 @@ def log_error_rack(exception, env, rack_filter)
 
       if should_send_email?
         # controller may not exist in some cases (like most 404 errors)
-        if controller
+        if (controller = exception_info.controller)
           controller.session["last_exception_timestamp"] = ExceptionHandling.last_exception_timestamp
```

Comment:
I don't think `ExceptionHandling` is needed here. It would be more DRY to omit it.